### DISCUSSION TITLE
Has the look and feel of this library been updated at all?

### DISCUSSION THREAD

**Discussion by adanvdo** [Krypton-Suite/Standard-Toolkit#626] [Q&A] [answered]
I am happy to see people are keeping this project going.  It's hard to find nice controls to use in open-source projects.  
However, looking at all of the control examples.. everything looks like it's from windows XP.  

I'm just curious if there have been any updates to the design of the controls to make them look more modern.  Any kind of feedback regarding that would be appreciated.

**Comment by Smurf-IV** [comment, marked as answer]
Switch to the "alpha" tree, and then go explore the pictures in the following location:
https://github.com/Krypton-Suite/Standard-Toolkit-Demos/tree/alpha

And even download the simple binaries installer, and have a "play" with the embedded themes.

There is also a "themes" github that has several  other flavours of UI.

Note: The hosting app, for demonstrating controls are normally not themed them selves

**Reply by adanvdo** [reply]
Thank you!
